deploy/sck/sck_147.yaml

```diff
@@ -38,57 +46,50 @@ splunk-kubernetes-logging:
       - release
       - environment
       - tier
+  # In case snmp prefix is useful or if you want to remove "kube"
+  sourcetypePrefix: "kube"
   splunk:
     hec:
       indexName: ##EVENTS_INDEX##
   logs:
-    #nginx-ingress:
-    #  from:
-    #    pod: nginx-ingress
-    #    container: nginx-ingress-controller
-    #  #keep default sourcetype for sake of avoiding complexity
-    #  #if you really want to customize sourcetype you can 
-    #  sourcetype: nginx:ingress:controller
-    #  multiline:
-    #    firstline: /^\w[0-1]\d[0-3]\d\s\d{2}:\d{2}:\d{2}.\d{6}\s|^\d+\.\d+\.\d+\.\d+[^\[]+|\d{4}\/\d{2}\/\d{2}\s\d{2}:\d{2}:\d{2}[^\[]+|^NGINX\sIngress\scontroller|^----------/
     sck:
       from:
         pod: sck-splunk-kubernetes-
         container: splunk-fluentd-k8s-
       multiline:
         firstline: /^\d{4}-\d{2}-\d{2}\s\d{2}\:\d{2}\:\d{2}\s\+\d{4}\s\[\w+\]\:/
         separator: "\n"
-        flushInterval: 2
+        flushInterval: 5
     
 #local config for objects chart
 splunk-kubernetes-objects:
+  # enable or diable objects
+  enabled: false
   rbac:
     create: true
   serviceAccount:
     create: true
     name: splunk-kubernetes-objects
   kubernetes:
     insecureSSL: true
-    clusterName: ##CUSTER_NAME##
   objects:
     core:
-      v1: [{"name":"pods", "interval":"60s"},{"name":"nodes", "interval":"60s"},{"name":"component_statuses", "interval":"60s"},{"name":"config_maps", "interval":"60s"},{"name":"namespaces", "interval":"60s"},{"name":"persistent_volumes", "interval":"60s"},{"name":"persistent_volume_claims", "interval":"60s"},{"name":"resource_quotas", "interval":"60s"},{"name":"services", "interval":"60s"},{"name":"service_accounts", "interval":"60s"},{"name":"events", "mode":"watch"},]
-    apps:
       v1:
-        [{"name":"daemon_sets", "interval":"60s"},{"name":"deployments", "interval":"60s"},{"name":"replica_sets", "interval":"60s"},{"name":"stateful_sets", "interval":"60s"},]
+        [{ "name": "pods", "interval": "300s" },{ "name": "nodes", "interval": "300s" },{ "name": "component_statuses", "interval": "300s" },{ "name": "namespaces", "interval": "300s" },{ "name": "services", "interval": "300s" },{ "name": "events", "mode": "watch" }]
```

Comment:
Can you please split this long like like my first comment above? This way the configuration would be much more readable.